im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
+    <author>
```

Comment:
no problem; there are a lot of specialty styles out there. That's why I wanted the documentation; see https://github.com/citation-style-language/styles/pull/1674/files#r38489949 on how to include it in the style.

Apart from that, there are only two remaining small bits ([here](https://github.com/citation-style-language/styles/pull/1674/files#r38465777) and [here](https://github.com/citation-style-language/styles/pull/1674/files#r38431529)) and we're ready to go.